blog/20240814-simplex-chat-vision-funding-v6-private-routing-new-user-experience.md

```diff
@@ -0,0 +1,183 @@
+
+[SimpleX Chat: vision and funding 2.0](#simplex-chat-vision-and-funding-20):
+- [The past](#the-past).
+- [The present](#the-present).
+- [The future](#the-future).
+
+[What's new in v6.0](#whats-new-in-v60):
+- [Private message routing](#private-message-routing) - now enabled by default
+- [New chat experience](#new-chat-experience)
+  - [Connect to your friends faster](#connect-to-your-friends-faster).
+  - [Archive contacts to chat later](#archive-contacts-to-chat-later).
+  - [New way to start chat](#new-way-to-start-chat).
+  - [Reachable chat toolbar](#reachable-chat-toolbar).
+  - [Moderate like a pro](#moderate-like-a-pro).
+  - [Increase font size](#increase-font-size)<sup>*</sup>.
+- [New media options](#new-media-options)
+  - [Play from the chat list](#play-from-the-chat-list).
+  - [Blur for better privacy](#blur-for-better-privacy).
+  - [Share from other apps](#share-from-other-apps)<sup>**</sup>.
+- [Improved networking and reduced battery usage](#improved-networking-and-reduced-battery-usage)
+
+\* Android and desktop apps.
+
+\*\* iOS app.
+
+## SimpleX Chat: vision and funding 2.0
+
+### The past
+
+...
+
+### The present
+
+...
+
+### The future
+
+...
+
+## What's new in v6.0
+
+v6.0 is one of our biggest releases ever, with a lot of focus on UX and stability improvements, and the new features the users asked for. 
+
+### Private message routing
+
+The private message routing [we announced before](./20240604-simplex-chat-v5.8-private-message-routing-chat-themes.md) is now enabled for all users by default – it protects users IP addresses and sessions from the destination servers.
+
+### New chat experience
+
+#### Connect to your friends faster 
+
+This version includes messaging protocol improvements that cut down the number of messages requried for two users to connect twice. Not only it means connecting faster and using less traffic, this change allows to start sending messages sooner than before - so you would see "connecting" in the list of the chats for a much shorter time than before.
+
+It will be improved further in the next version - you will be able to send messages straight after using the link, without waiting for your contact to be online.
+
+#### Archive contacts to chat later
+
+Now you have two new options when deleting a conversation:
+- only delete conversation, and archive contact - we will add archiving conversation without clearing it in the next version.
+- delete contact but keep the conversation.
+
+Also, deleting a contact now requires double confirmation, so you are less likely to delete the contact accidentally.
+
+#### New way to start chat
+
+Now when you tap pencil button, you will see a large New message sheet, that adds to the options you had before some new functions.
+
+Old options:
+- *Add contact* to create a new 1-time invitation link,
+- *Scan/paste link*: to use the link you received, it can be 1-time invitation, a public SimpleX address, or a link to join the group.
```

Comment:
```suggestion
- *Scan/Paste link*: to use the link you received, it can be 1-time invitation, a public SimpleX address, or a link to join the group.
```